DemoApp/Assets/Plugins/Android/AndroidManifest.xml

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
nit: might want to just open this file in AS and run the formatter.

Reply:
Unity reverts the formatting when exporting the project so it's not really worth it imo.

Reply:
We don't really interact with the exported file, but we do with this one. When we want to edit this file in the future, I feel like it will be easier to do so, if it is properly formatted

Reply:
Makes sense, updated formatting.